source.go

```diff
@@ -1,14 +1,14 @@
 package config
 
 // Getter gets a value for a key.
-type Getter func(key string) (string, error)
+type Getter func(tag tagValue) (string, error)
```

Comment:
Should `tagValue` be an exported type? It is used by the exported interface `Source`.